stubs/tests/ResourceCollectionTest.php

```diff
@@ -0,0 +1,105 @@
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\App;
+use Tests\TestCase;
+
+class ResourceCollectionTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_resource_collections_can_be_translated()
+    {
+        if (! config('hearth.resources.enabled')) {
+            return $this->markTestSkipped('Resource support is not enabled.');
+        }
+
+        $resourceCollection = ResourceCollection::factory()->create();
+
+        $titleTranslations = ['en' => 'title in English', 'fr' => 'title in French'];
+        $descriptionTranslations = ['en' => 'description in English', 'fr' => 'description in French'];
+
+        $resourceCollection->setTranslation('title', 'en', $titleTranslations['en']);
+        $resourceCollection->setTranslation('title', 'fr', $titleTranslations['fr']);
+
+        $resourceCollection->setTranslation('description', 'en', $descriptionTranslations['en']);
+        $resourceCollection->setTranslation('description', 'fr', $descriptionTranslations['fr']);
+
+        $this->assertEquals($titleTranslations['en'], $resourceCollection->title);
+        $this->assertEquals($descriptionTranslations['en'], $resourceCollection->description);
+        App::setLocale('fr');
+        $this->assertEquals($titleTranslations['fr'], $resourceCollection->title);
+        $this->assertEquals($descriptionTranslations['fr'], $resourceCollection->description);
+
+        $this->assertEquals($titleTranslations['en'], $resourceCollection->getTranslation('title', 'en'));
+        $this->assertEquals($descriptionTranslations['en'], $resourceCollection->getTranslation('description', 'en'));
+        $this->assertEquals($titleTranslations['fr'], $resourceCollection->getTranslation('title', 'fr'));
+        $this->assertEquals($descriptionTranslations['fr'], $resourceCollection->getTranslation('description', 'fr'));
+
+        $this->assertEquals($titleTranslations, $resourceCollection->getTranslations('title'));
+        $this->assertEquals($descriptionTranslations, $resourceCollection->getTranslations('description'));
+
+        $this->expectExceptionMessage("Cannot translate attribute `user_id` as it's not one of the translatable attributes: `title, description`");
+        $resourceCollection->setTranslation('user_id', 'en', 'user_id in English');
+    }
+
+    public function test_resource_collections_belong_to_user_get_deleted_on_user_delete()
+    {
+        $resourceCollection = ResourceCollection::factory()->create();
+
+        $resourceCollection->user->delete();
+        $this->assertModelMissing($resourceCollection);
+    }
+
+    public function test_single_user_can_have_multiple_resource_collections()
+    {
+        $user = User::factory()->create();
+        $resourceCollection = ResourceCollection::factory(5)
+            ->for($user)
+            ->create();
+        $this->assertCount(5, ResourceCollection::where('user_id', $user->id)->get());
+    }
```

Comment:
I'm thinking that this should probably be in the user tests. Also, in the assertion, can you get the ResouceCollections by calling `$user-> resourceCollections()` that way you can test the method.

Reply:
I see that we don't have a test file specifically for User, should I create one and include this?

Reply:
I see there isn't one in the stubs. I guess you could either add it here because there is a change to the User model, or file a new issue to write tests for the User model.

Reply:
Since we don't have specific test files for all the models, like Invitation, Membership and User, I think it makes sense to keep this test case in ResourceCollections for now. Models without their own test file are still tested as a part of other test cases.